tests/test-javalin-jsonb/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
oh yeah, I don't think we want this here. We have a LICENSE file at the top of the project and that is expected to cover all content in the project. IMO it's a negative to add extra LICENSE files in sub-directories.